meta-mentor-staging/recipes-graphics/mesa/mesa-demos_8.2.0.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend_amd := "${THISDIR}/${PN}_${PV}:"
```

Comment:
What's with the `_${PV}` here? We never use an underscore separator for the filesdir subdirectory, ever. Either use `${PN}`, or use `${PN}-${PV}`.